install/packages.rst

```diff
@@ -184,6 +184,21 @@ download-opkg
 Downloads an ipk file from the RobotPy and NI's online opkg repositories, along
 with the dependencies for the package.
 
+You can also use a `requirements.txt` file to specify which packages should be downloaded:
+::
```

Comment:
Having a colon at the end of the previous line and this double-colon on its own line is a bit redundant.

Reply:
Didn't realize that the double colon would also add a colon to the previous line. RST is a bit weird sometimes.